src/gpustat/usr/local/emhttp/plugins/gpustat/lib/Nvidia.php

```diff
@@ -113,9 +113,19 @@ private function detectApplication (SimpleXMLElement $process)
                             }
                         }
                     }
-                    $this->pageData[$app . 'using'] = true;
```

Comment:
These would still be required for non multi i.e. pre 6.12 unless same changes are applied to the pages and js as this common for both.

Reply:
I also updated code for pre 6.12

Reply:
nchan can only be used if os is 6.10 or greater. Not sure if there will be users running pre 6.10 Non multy should use ajax updates for backwards compatibilty.

Reply:
I will revert it and use ajax polling for non multigpu. Sorry.

Reply:
I reverted to ajax polling for non multi.